central/reportconfigurations/service/service_impl_test.go

```diff
@@ -30,11 +30,11 @@ type TestReportConfigurationServiceTestSuite struct {
 }
 
 func (s *TestReportConfigurationServiceTestSuite) SetupTest() {
+	s.mockCtrl = gomock.NewController(s.T())
 	if env.PostgresDatastoreEnabled.BooleanSetting() {
```

Comment:
use the new pgtest function?

Reply:
sort of trying to minimize changes tbh. This will eventually be all removed when pg flag is removed